docs/index.html

```diff
@@ -217,6 +217,9 @@ <h2>Useful information</h2>
 										<strong>For Fedora users,</strong> you may check that you have installed the library <a href="https://pkgs.org/search/?q=librtmp" target="_blank">librtmp</a>.
 									</p>
 
+									<p>
+										<strong>For Debian or Ubuntu users,</strong> you may set an environment variable <code>LIBVA_DRIVER_NAME=iHD</code> or start the AppImage like this <code>LIBVA_DRIVER_NAME=iHD ./Shadow.AppImage --no-sandbox</code>.
+									</p>

```

Comment:
Maybe add "for recent Intel GPU" because older ones should stick with `i965` driver

Reply:
It's better like this ?

Reply:
Perfect :)